native/components/Text.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components/native";
+
+const Text = styled.Text`
+  font-size: 18px;
+  line-height: 28px;
+`;
+
+/** @component */
+export default Text;
```

Comment:
May want to switch this to a named export. In `stellar/stellar.org` I've been putting all of the 'text' elements [in a `Text.js`](https://github.com/stellar/stellar.org/blob/master/src/basics/Text.js) as named exports, since there are a bunch and they don't have much code per-component.

Reply:
`react-styleguidist` has a bug where example files get the default export instead of the named exports:
https://github.com/styleguidist/react-styleguidist/issues/1281

So the solution was to switch to default exports:
https://github.com/stellar/js-stellar-elements/pull/5/commits/5f19d02152c5735eedf709b35c1a2f4833724309

It's not great, but it seems fine to localize this rule-breaking to one repo only.